src/functionalTest/resources/scenarios/RIA-8969-progress-migrated-case-desired-state.json

```diff
@@ -9,7 +9,6 @@
       "caseData": {
         "template": "minimal-appeal-started.json",
         "replacements": {
-          "isAdmin": "Yes",
```

Comment:
Why IsAdmin should be deleted? for https://github.com/hmcts/ia-case-notifications-api/pull/1286/files you are setting isAdmin to Yes

Reply:
Hi Miguel,
This functional test was for the event and was breaking after adding the notification.
The options were to add more case details, so the notification steps were satisfied, or delete the isAdmin part.
I went with deleting as they are not used in the event part of the code.
Thanks

Reply:
I would say to go with satisfying the notification adding the missing fields, the reason is that isAriaMigrated case will always been isAdmin = "Yes" as well

Reply:
Update with listCaseHearingCentre